iati/validator.py

```diff
@@ -533,6 +413,36 @@ def full_validation(dataset, schema):
     return error_log
 
 
+def get_error_codes():
```

Comment:
Whilst this there is some test coverage for this would be tempted to add a test for this function - namely to check that the structure of the returned `dict` is correct, including an expected base_exception that is actually an exception...

Reply:
This exists at https://github.com/IATI/iati.core/blob/2f436cbace0d8e451bcd0064d46b0944ed10e575/iati/tests/test_validator.py#L225-L238

Reply:
:+1: Nice - missed that.